### ISSUE TITLE
exe doesn't open in Windows 10

### ISSUE THREAD

**Issue by austinmroczek** [achhabra2/kqb-json-viewer#1]
Downloaded v0.0.4, unzipped and double clicked on the exe.  No window opens.  No visible error message.  No windows log messages.

```txt
Edition	Windows 10 Home
Version	20H2
OS build	19042.804
Experience	Windows Feature Experience Pack 120.2212.551.0
```

Also tried running from command line.  No window opens.  No error message in command prompt.

**Comment by ianfitzpatrick** [comment]
I think the game doesn't handle the state where you haven't played any matches. I ran into this when I tried to run it on my Mac too. I hadn't played any games on there since the new patch came out, when these log files started generating. Once I played one custom to finish, it then worked.

So it just needs to handle this case, assuming it's the same issue.

**Comment by austinmroczek** [comment]
There was zero state for me since I don't have the game on PC.  Was trying to work with files from teammates

**Comment by austinmroczek** [comment]
This is corrected, however see #3 about the need to display an error message in the GUI.
